kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java

```diff
@@ -120,8 +121,8 @@
     private final GwtNetworkServiceAsync gwtNetworkService = GWT.create(GwtNetworkService.class);
     private final GwtDeviceServiceAsync gwtDeviceService = GWT.create(GwtDeviceService.class);
 
-    private static final String REGEX_PASSWORD_WPA = "^[ -~]{8,63}$";
-    private static final String REGEX_PASSWORD_WEP = "^(?:[\\x00-\\x7F]{5}|[\\x00-\\x7F]{13}|[a-fA-F0-9]{10}|[a-fA-F0-9]{26})$";
+    private static final String REGEX_PASS_WPA = "^[ -~]{8,63}$";
+    private static final String REGEX_PASS_WEP = "^(?:[\\x00-\\x7F]{5}|[\\x00-\\x7F]{13}|[a-fA-F0-9]{10}|[a-fA-F0-9]{26})$";
```

Comment:
Why do we need to change those names?

Reply:
Sonar complains about the use of a constant with the name PASSWORD...

Reply:
Oh, ok.